src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility2.cs

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// AccessibilityState2 is an enumeration that represents the states of the view to send to accessibility service.
+    /// </summary>
+    public enum AccessibilityState2
+    {
+        /// <summary>
+        /// Indicates whether the view is enabled or not.
+        /// </summary>
+        Enabled = 0,
+        /// <summary>
+        /// Indicates whether a selectable element is currently selected or not.
+        /// </summary>
+        Selected,
+        /// <summary>
+        /// Indicates the state of a checkable element.
+        /// </summary>
+        Checked,
+        /// <summary>
+        /// Indicates whether an element is currently busy or not.
+        /// </summary>
+        Busy,
+        /// <summary>
+        /// Indicates whether an expandable element is currently expanded or collapsed.
+        /// </summary>
+        Expanded
+    }
+
+    /// <summary>
+    /// The AccessibilityStates2 structure represents a set of states of the view to communicate to accessibility service.
+    /// </summary>
+    public struct AccessibilityStates2
+    {
+        const uint _on = 1;
+        uint _bitMask;
+
+        /// <summary>
+        /// Gets the state at the specsified name
+        /// </summary>
+        /// <param name="state">The name of the state</param>
+        /// <returns>The state af the specified name</returns>
+        public bool this[AccessibilityState2 state]
+        {
+            get
+            {
+                return Convert.ToBoolean(_bitMask & (_on << (int)state));
+            }
+            set
+            {
+                if (value)
+                {
+                    _bitMask |= (_on << (int)state);
+                }
+                else
+                {
+                    _bitMask &= ~(_on << (int)state);
+                }
+            }
+        }
+
+        AccessibilityStates2(int states)
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
        public AccessibilityStates2(int states)
```

I don't know how we define to struct's constructor level but.. It looks browsable  never required here.